api/manifestInjector.go

```diff
@@ -26,6 +26,17 @@ func InjectSteps(preferences *manifest.EstafetteManifestPreferences, mft manifes
 		AutoInjected:   true,
 	}
 
+	snykStep := &manifest.EstafetteStage{
+		Name:           "snyk-scan",
+		ContainerImage: fmt.Sprintf("extensions/snyk-scan:%v", builderTrack),
```

Comment:
the image location is: **eu.gcr.io/travix-com/snyk-scanner**; do we need to push it to **extensions/snyk-scan** ?

Reply:
I think so. @vitorarins  Do you know if there is any restriction regarding the location of the extension's image?

Reply:
The only restriction is that image should be public, which is not the case with `eu.gcr.io/travix-com/snyk-scanner`.
Don't forget this is an open source project. Let's create `estafette-extension-snyk-scanner` repo in github and publish it to Docker Hub.

Reply:
Right. Thanks @vitorarins. I will move the repo and image.

Reply:
I am with Vitor in this sense, but actually I'd prefer to be the logic in this repo in golang not by simple bash file